test/flutter_checks.dart

```diff
@@ -74,6 +74,18 @@ extension ValueListenableChecks<T> on Subject<ValueListenable<T>> {
   Subject<T> get value => has((c) => c.value, 'value');
 }
 
+extension ThemeDataChecks on Subject<ThemeData> {
```

Comment:
nit: for logical ordering, put these Material-specific items down among the other Material-specific items near the end of the file